crates/bevy_ecs/src/system/query.rs

```diff
@@ -1082,6 +1082,94 @@ impl<'w, 's, D: QueryData, F: QueryFilter> Query<'w, 's, D, F> {
         }
     }
 
+    /// Returns a parallel iterator over the read-only query items generated from an [`Entity`] list.
+    ///
+    /// Entities that don't match the query are skipped. Iteration order and thread assignment is not guaranteed.
+    ///
+    /// If the `multithreaded` feature is disabled, iterating with this operates identically to [`Iterator::for_each`]
+    /// on [`QueryManyIter`].
+    ///
+    /// This can only be called for read-only queries, there is no `par_iter_many_mut` equivalent.
```

Comment:
/// This can only be called for read-only queries, there is no `par_iter_many_mut` equivalent.
```suggestion
    /// This can only be called for read-only queries, there is no `par_iter_many_mut` equivalent to avoid potential aliasing
```